### PR TITLE
JDK-8287094: IGV: show node input numbers in edge tooltips

### PR DESCRIPTION
**Pull request by tobiasholenstein**
For nodes with many inputs, such as safepoints, it is difficult and error-prone to figure out the exact input number of a given incoming edge. 

Extend the Edge Tooltips to include the input number of the destination node:
**Before** `91 Addl -> 92 SafePoint`
**Now** `91 Addl -> 92 SafePoint [NR]`
edge

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8287094](https://bugs.openjdk.org/browse/JDK-8287094): IGV: show node input numbers in edge tooltips

### Reviewers
 * [Christian Hagedorn](https://openjdk.org/census#chagedorn) (@chhagedorn - **Reviewer**)
 * [Tobias Hartmann](https://openjdk.org/census#thartmann) (@TobiHartmann - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/9273/head:pull/9273` \
`$ git checkout pull/9273`

Update a local copy of the PR: \
`$ git checkout pull/9273` \
`$ git pull https://git.openjdk.org/jdk pull/9273/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 9273`

View PR using the GUI difftool: \
`$ git pr show -t 9273`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/9273.diff">https://git.openjdk.org/jdk/pull/9273.diff</a>

</details>

### CODE REVIEW

File: src/utils/IdealGraphVisualizer/Graph/src/main/java/com/sun/hotspot/igv/graph/FigureConnection.java [hunk 1]
```diff
@@ -105,6 +105,7 @@ public String getToolTipText() {
         builder.append(getOutputSlot().getFigure().getProperties().resolveString(shortNodeText));
         builder.append(" → ");
         builder.append(getInputSlot().getFigure().getProperties().resolveString(shortNodeText));
+        builder.append(" [" + getInputSlot().getPosition() + "]");
```
Comment (chhagedorn) [inline src/utils/IdealGraphVisualizer/Graph/src/main/java/com/sun/hotspot/igv/graph/FigureConnection.java:108]: String concatenation could be replaced by `append()` calls to follow the pattern above (the `builder.` can also be removed and the calls be chained together directly):
```suggestion
        builder.append(" [")
               .append(getInputSlot().getPosition())
               .append("]");
```

**Review by chhagedorn — APPROVED** [review]
Looks good!

**Review by TobiHartmann — APPROVED** [review]
Looks good.

**Review by chhagedorn — APPROVED** [review]

**Comment by tobiasholenstein** [comment]
@chhagedorn and @TobiHartmann thanks for the reviews!

**Pull request closed without merging**